writeit/settings.py

```diff
@@ -67,6 +67,10 @@
 # If you set this to False, Django will not use timezone-aware datetimes.
 USE_TZ = True
 
+# If the site is being served over https change this to 'https' so that links
+# to subdomains use the correct scheme.
+DEFAULT_URL_SCHEME = 'http'
+
```

Comment:
Looks like it's also used in [django-subdomains `reverse`](https://github.com/tkaemming/django-subdomains/blob/db7f08db7586ba7b4f713aebb83daf2621edde6b/subdomains/utils.py#L62), which we use instead of Django's standard `reverse`. So I think we will need to change this to correctly link to subdomain?